packages/@aws-cdk/aws-cognito/lib/user-pool-client.ts

```diff
@@ -0,0 +1,53 @@
+import cdk = require('@aws-cdk/cdk');
+import { CfnUserPoolClient } from './cognito.generated';
+import { UserPool } from './user-pool';
+
+export enum AuthFlow {
+  ADMIN_NO_SRP_AUTH = 'ADMIN_NO_SRP_AUTH',
```

Comment:
I'd be in favor of using `PascalCase` names here, but I also don't mind this `SHOUT_CASE`.

Reply:
Me too, I just used SHOUT_CASE out of habit :). I'll switch the other enums to PascalCase as well.

Reply:
Me too, I just went with `SHOUT_CASE` out of habit :). I'll update the other enums as well.